html/includes/api_functions.inc.php

```diff
@@ -825,6 +825,30 @@ function list_available_health_graphs()
                 'name' => 'device_'.$graph['sensor_class'],
             );
         }
+
+        $processor = dbFetchRows('SELECT * FROM `processors` WHERE device_id = ?', array($device_id));
```

Comment:
Shouldn't these be count queries? `SELECT COUNT(*) ...`

Reply:
That would be fine by me, all this needs to be converted to Eloquent eventually.  Eloquent is safe to use in API and Web UI.  It is disable in poller/discovery for now until we can send a notification for uses to increase their mysql max connections.